ngboost/distns/normal.py

```diff
@@ -31,12 +31,23 @@ def D_nll(self, Y_):
 
     def crps(self, Y):
         Z = (Y - self.loc) / self.scale
-        return self.scale * (Z * (2 * sp.stats.norm.cdf(Z) - 1) + \
-               2 * sp.stats.norm.pdf(Z) - 1 / np.sqrt(np.pi))
+        return (self.scale * (Z * (2 * sp.stats.norm.cdf(Z) - 1) + \
+                2 * sp.stats.norm.pdf(Z) - 1 / np.sqrt(np.pi)))
+
+    def D_crps(self, Y_):
+        Y = Y_.squeeze()
+        Z = (Y - self.loc) / self.scale
+        D = np.zeros((self.var.shape[0], 2))
+        D[:, 0] = -(2 * sp.stats.norm.cdf(Z) - 1)
+        D[:, 1] = self.crps(Y) + (Y - self.loc) * D[:, 0]
+        return D
```

Comment:
This looks correct. What about censoring?

Reply:
Haven't thought about the censored case yet.

[Somewhat unrelated] How do you feel about keeping Normal restricted to Regression problems and LogNormal restricted to Survival problems? Then we can keep the censored `D_crps` in `lognormal.py`.